db/migrate/20141226232633_create_trucking_drivers_views.rb

```diff
@@ -1,14 +1,18 @@
 class CreateTruckingDriversViews < ActiveRecord::Migration
   using(:grossman)
 
-  def change
-    create_view :Trucking_Drivers, '
-SELECT
-  driver_number AS DriverID
-, first_name AS FirstName
-, last_name AS LastName
-, driver_type AS DriverType
-FROM
-  drivers'
+  def up
```

Comment:
I assume that this still doesn't get included in the schema file (sql or otherwise)?